_episodes/05-dockerfiles.md

```diff
@@ -250,7 +273,7 @@ way to bring them into the Docker build.
 [docker-docs-FROM]: https://docs.docker.com/engine/reference/builder/#from
 [docker-docs-build-arg]: https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg
 [docker-docs-ENV]: https://docs.docker.com/engine/reference/builder/#env
-[docker-docs-tag]: https://docs.docker.com/engine/reference/commandline/tag/
+[podman-docs-tag]: https://docs.podman.io/en/latest/markdown/podman-tag.1.html
 [docker-docs-COPY]: https://docs.docker.com/engine/reference/builder/#copy
```

Comment:
a lot of these links are still to Docker docs.

Reply:
Some of them refer to the `Dockerfile` documentation. Others are not used anymore in the body of the chapter, so I am removing them